ansible_wisdom/ai/api/model_client/wca_utils.py

```diff
@@ -161,6 +178,7 @@ def __init__(self):
                 ResponseStatusCode403UserTrialExpired(),
                 ResponseStatusCode403(),
                 ResponseStatusCode404WCABadRequestModelId(),
+                ResponseStatusCode404(),
```

Comment:
Do we also need to check for the unknown 404 for `contentmatch`: https://github.com/ansible/ansible-ai-connect-service/blob/baf482d6490ddda9c09aa373dcf3d904d65544a1/ansible_wisdom/ai/api/model_client/wca_utils.py#L198

Reply:
We should... I thought about it but didn't because it was bigger than just adding it to the contentmatch list since it raises a WcaInferenceFailure, which wouldn't be right. I was thinking perhaps we should have a generic check at the end that for any non-200 logs some details and raises a base WcaException. In a future iteration... :)